api/src/main/java/openconsensus/tags/unsafe/ContextUtils.java

```diff
@@ -43,5 +49,18 @@ private ContextUtils() {}
       Context.keyWithDefault("openconsensus-tag-map-key", EMPTY_TAG_MAP);
 
   @Immutable
-  private static final class EmptyTagMap extends TagMap {}
+  private static final class EmptyTagMap extends TagMap {
```

Comment:
Why a second `EmtpyTagMap`? Any special reason?

Reply:
See comment in the description.